### ISSUE TITLE
When will the severe drift issue with wireless be fixed? 

### ISSUE THREAD

**Issue by PolaCat** [pushrax/OpenVR-SpaceCalibrator#80]
This is what I am currently going through and it is a living hell to do anything in, i just want to know if anyone is also having this issue

it's just serve and awful drift, I just want to know if anyone else has the same issue

**Comment by Blu3w0l1f** [comment]
yeah I have the same issue on my 3.0 vive trackers the hip one will float away and it happens quite a bit
